app-backend/build.gradle.kts

```diff
@@ -1,11 +1,11 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
Why do you refactor this back to 2 spaces identation? There is no editorconfig with this information and 4 spaces is more common setup AFAIK

Reply:
fixed.